### DISCUSSION TITLE
Completely disable default menubar?

### DISCUSSION THREAD

**Discussion by JaceThings** [FelixKratz/SketchyBar#643] [Q&A]
Is it possible to completely disable the default macOS menu buyer so that it is never interact or visible whenever the mouse enters the top part of the screen, basically completely disabling it.

**Comment by icecmach** [comment]
It seems the menu is controlled by the process SystemUIServer but when I try to kill it, it will auto restart.

**Comment by Pe8er** [comment]
Yabai can do it: https://github.com/koekeishiya/yabai/issues/2091

**Comment by Bellavene** [comment]
https://github.com/aspauldingcode/macwmfx
